### DISCUSSION TITLE
How are the "Automatic updates" working?

### DISCUSSION THREAD

**Discussion by mordax7** [pdulvp/jellyfin-qnap#107] [General]
I'd like to understand how the automatic updates work. As shown in the image below, I currently have Jellyfin version 10.9.10 installed, but I noticed that version [10.9.11](https://github.com/pdulvp/jellyfin-qnap/releases/tag/10911.10911.6018.5143_ec4056cb) is available in the [Jellyfin QNAP store](https://pdulvp.fr/jellyfin-qnap.html). However, as seen in the image, I don't see any update button. 

Why is that? Would it be better to update manually?

![image](https://github.com/user-attachments/assets/5e076d62-b033-4e72-a5a5-f0c0bd8073be)

**Comment by sankukai44** [comment]
Hi,

I think the update is here, i just dit it, you should be able to.

**Reply by mordax7** [reply]
This would prove my theory.

Most likely the store update is available only after [this repo](https://github.com/pdulvp/qnap-store) updates the repos.xml file.
The .11 update was already released a few weeks ago but the Qnap Store update came just two days ago.

I still wonder when the bot actually runs and [updates the store](https://github.com/pdulvp/qnap-store/commit/adc765d1dc00364edd92b2eb85d31553438b638f).

**Reply by pdulvp** [reply]
The bots are launched manually (sic) when I think of it, 
The store is theorically updated after a release as been published several days on github releases. 
(to avoid wide spread of a bugged version)

**Reply by mordax7** [reply]
@pdulvp Would it be possible to have a repository which updates automatically as soon there is a new release?

**Reply by pdulvp** [reply]
Hi, 
You can add the "Pre-release" url from https://github.com/pdulvp/qnap-store?tab=readme-ov-file#qnap-store
I will mark the new releases as 'pre-release' for a while then.

**Reply by mordax7** [reply]
@pdulvp hey. Sounds good :)

**Reply by ahasenfr** [reply]
Hi together :)
I have installed Version 10.10.1-01 just before 02 came out.
Then I thought okay, I will try the automatic update.

Added the repo as described:
![image](https://github.com/user-attachments/assets/088ca0d5-d9f0-483f-a9f8-f8d726dea4f8)
But I didn't add any login credentials.

I have the settings in the form that only necessary updates are done automatically.
So I went in and wanted to install it via the app manager manually.
I have even shut down the jellyfin application beforehand.
But still it doesn't work.

Before clicking install:
![image](https://github.com/user-attachments/assets/81917238-0e73-4c29-bf23-1b4213a41ac1)

After the install process had been started:
![image](https://github.com/user-attachments/assets/f1d7eb96-b8ce-4815-a618-9d6e9b31577d)
it says i haven't any apps installed.
refreshing brings back the apps.

so what went wrong`?

EDIT:
just had a look in the notifications:
![image](https://github.com/user-attachments/assets/0717db24-6dbf-4f24-ac94-25e2e8089c9e)

**Reply by pdulvp** [reply]
Hi, 
I think it's because you installed manually first.
You have to do some manual stuff to reconnect the store (rather than a whole install/uninstall)
https://github.com/pdulvp/jellyfin-qnap/discussions/115#discussioncomment-11208599
